### PR TITLE
Shayan/Feature market selector new design

### PR DESCRIPTION
**Pull request by shayan-binary-2**

### CODE REVIEW

File: sass/components/_search.scss [hunk 1]
```diff
@@ -0,0 +1,85 @@
+.sc-search-input {
+    width: 100%;
+    padding: 0px;
+    box-sizing: border-box;
+    position: relative;
+
+    > input {
+        padding-left: 32px;
+        line-height: 2em;
```
Comment (zaki-binary) [inline sass/components/_search.scss:9]:
```suggestion
        line-height: 2;
```

File: sass/components/_search.scss [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        font-size: 1.4em;
```
Comment (zaki-binary) [inline sass/components/_search.scss:10]:
```suggestion
        font-size: 1.4rem;
```

File: sass/components/_search-input.scss [hunk 3]
```diff
@@ -0,0 +1,85 @@
+.sc-lookup-input {
+    width: 100%;
+    padding: 0px;
+    box-sizing: border-box;
+    position: relative;
+
+    > input {
+        padding-left: 32px;
+        line-height: 2em;
```
Comment (zaki-binary) [inline sass/components/_search-input.scss:9]:
```suggestion
        line-height: 2;
```

File: sass/components/_search-input.scss [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        font-size: 1.4em;
```
Comment (zaki-binary) [inline sass/components/_search-input.scss:10]:
```suggestion
        font-size: 1.4rem;
```

**Comment by shayan-binary-2** [comment]
@zaki-binary I should later do a refactor to apply this in all CSS files.

File: sass/components/_search-input.scss (hunk 3 above)
Comment (shayan-binary-2) [reply sass/components/_search-input.scss]: this file totally removed

File: sass/components/_search-input.scss (hunk 4 above)
Comment (shayan-binary-2) [reply sass/components/_search-input.scss]: this file totally removed

File: sass/components/_search.scss (hunk 1 above)
Comment (shayan-binary-2) [reply sass/components/_search.scss]: applied this

File: sass/components/_search.scss (hunk 2 above)
Comment (shayan-binary-2) [reply sass/components/_search.scss]: applied this

**Review by zaki-binary — APPROVED** [review]

**Pull request merged**